pkg/common/cns-lib/volume/manager.go

```diff
@@ -2938,3 +2966,21 @@ func GetAllManagerInstances(ctx context.Context) map[string]*defaultManager {
 	}
 	return newManagerInstanceMap
 }
+
+func (m *defaultManager) getAggregatedSnapshotSize(ctx context.Context, volumeID string) (int64, error) {
```

Comment:
check if int64 is sufficient

Reply:
int64 will be enough, aggregated size returned from cns is int64